File: CS/WpfApplication1/Data/CustomClasses.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WpfApplication1 {
    public class Country {
        public int Id { get; set; }
        public string Name { get; set; }
        public PopulationStats Stats { get; set; }
    }

    public class Person {
        public int Id { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
    }


    public class Address {
        public int Number { get; set; }
        public string Name { get; set; }
        public Country Country { get; set; }
        public List<Person> Persons { get; set; }

    }

    public class PopulationStats {
        public int Id { get; set; }
        public int CitizensCount { get; set; }
    }
}

File: VB/WpfApplication1/Data/CustomClasses.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace WpfApplication1
	Public Class Country
		Public Property Id() As Integer
		Public Property Name() As String
		Public Property Stats() As PopulationStats
	End Class

	Public Class Person
		Public Property Id() As Integer
		Public Property Firstname() As String
		Public Property Lastname() As String
	End Class


	Public Class Address
		Public Property Number() As Integer
		Public Property Name() As String
		Public Property Country() As Country
		Public Property Persons() As List(Of Person)

	End Class

	Public Class PopulationStats
		Public Property Id() As Integer
		Public Property CitizensCount() As Integer
	End Class
End Namespace
